slackBot.js

```diff
@@ -107,6 +107,15 @@ class SlackBot {
       this.slackEvents.stop(slackEventsPort);
       this.slackInteractiveMessages.stop(slackInteractiveMessagesPort);
     }
+
+    // can add a custom command
+    addCustomCommand(command, callback){
```

Comment:
this is a bit duplicated from `setCommandListener`, it's doing the exact same thing

Reply:
setCommandListener is set to only listen for '/' or '~' prefix this way they can listen to any type of command prefix they want. also if the programmer doesnt want to interact with slacks command page to set up commands they can use a different prefix. this one a developer can do something like discordbot.addCustomCommand('!test', (command) => { custom code to handle command}) let me know if not i can just remove it.

Reply:
There's a feeling from my end is that the slack event api isn't really made for the command listener. That's why when I'm trying to type anything starts with '/' the slack app doesn't allow me to type the command hasn't been defined on the app setting. 
I found a more high level slack library https://www.npmjs.com/package/@slack/bolt, it's using the event api to react to different event instead of command.
Anyway, it doesn't hurt adding this function if you feel it's necessary for the developer